src/main/java/models/GameEngine.java

```diff
@@ -40,6 +40,48 @@ public void initPlayer() {
             p1.pickupCard();
             p2.pickupCard();
         }
+
+    }
+
+    public void endGame(){
```

Comment:
Hur ska endGame() användas utifrån?
Alla if satser för att kolla om någon har slut på life görs inne i endGame() utan metoder..

Förslag om du inte kommer på nåt annat:
endGame() kan döpas till checkHealth() eller något som kan köras efter varje spelrunda..

Vi har inte fått in gameLogic branchen i dev än, men när den är inne kommer där va en boolean playing,
den kan du sätta till false i alla if satserna, så går den ur inre gameloopen